pkg/client/keycloak/client.go

```diff
@@ -291,20 +291,16 @@ func (kc *kcClient) IsSameOrg(client *gocloak.Client, orgId string) bool {
 		return false
 	}
 	attributes := *client.Attributes
-	return attributes["rh-org-id"] == orgId
+	return attributes[OrgKey] == orgId
 }
 
 func (kc *kcClient) IsOwner(client *gocloak.Client, userId string) bool {
 	if userId == "" {
 		return false
 	}
 	attributes := *client.Attributes
-	if rhUserId, found := attributes["rh-user-id"]; found {
-		if rhUserId == userId {
-			return true
-		}
-	} else {
-		return true
+	if rhUserId, found := attributes[UserKey]; found {
+		return rhUserId == userId
 	}
```

Comment:
I believe we removed too much. According to previous code, if the key is not found, it should return `true`.
If removing this else branch we don't have any test failing, I believe we are missing one test case :)

Reply:
should it not return false?

Reply:
That's a good question. Before it was doing 
```go
if rhUserId, found := attributes["rh-user-id"]; found {
		if rhUserId == userId {
			return true
		}
	} else {
		return true
```

so when `found` was false, was returning `true`. Now when `found` is `false` it returns `false`.
@akoserwal What is the correct behaviour here? The old one or the one introduced with this PR?

Reply:
`else { return true } ` was added because some of the initial service accounts didn't have the rh-user-id claims. Now it can be removed. What @pawelpaszki added in correct going forward